frontend/app/components/forms/signup-form.js

```diff
@@ -128,6 +128,11 @@ export default Component.extend({
         $('#feedback').html('');
       }
     }
+    if ($('#pwd').val() != $('#pwdConfirm').val()) {
```

Comment:
This Jquery can be avoided if you rely on the KeyUp event and propagate the state up to parent component and mutate the variable in the component to change the CSS. This can be achieved without the help of JQuery in the first place.

Reply:
Done :+1: